index.html

```diff
@@ -88,25 +91,143 @@
 							<ul>
 								<li><a href="UmbracoCms.4.5.0.zip" rel="download">4.5.0</a> - .NET 4.0; dropped support for VistaDB. <a href="utilities/VistaDBtoSQL.zip" title="VistaDB to SQL Server">Need to migrate to SQL?</a></li>
 								<li><a href="UmbracoCms.4.5.1.zip" rel="download">4.5.1</a></li>
-								<li><a href="UmbracoCms.4.5.2.zip" rel="download">4.5.2</a></li>
+								<li>
+									<a href="UmbracoCms.4.5.2.zip" rel="download">4.5.2</a>
+									<span class="commit">
+										(<a href="https://github.com/umbraco/Umbraco-CMS/releases/tag/Release-4.5.2" rel="external" target="_blank">See release on GitHub</a>)
+									</span>
+								</li>
+								<li>
+									4.6.0
+									<span class="commit">
+										(<a href="https://github.com/umbraco/Umbraco-CMS/releases/tag/Release-4.6.0" rel="external" target="_blank">See release on GitHub</a>)
+									</span>
+								</li>
 								<li><a href="UmbracoCms.4.6.1.zip" rel="download">4.6.1</a> - v4.6.0 had a very short release</li>
-								<li><a href="UmbracoCms.4.7.0.zip" rel="download">4.7.0</a></li>
+								<li>
+									<a href="UmbracoCms.4.7.0.zip" rel="download">4.7.0</a>
+									<span class="commit">
+										(<a href="https://umbraco.codeplex.com/SourceControl/changeset/view/6f33e2b81175" rel="external" target="_blank">See commit 6f33e2b81175 on CodePlex</a>)
+									</span>
+								</li>
 								<li><a href="UmbracoCms.4.7.1.zip" rel="download">4.7.1</a></li>
 								<li><a href="UmbracoCms.4.7.1.1.zip" rel="download">4.7.1.1</a></li>
-								<li><a href="UmbracoCms.4.7.2.zip" rel="download">4.7.2</a></li>
-								<li><a href="UmbracoCms.4.8.0.zip" rel="download">4.8.0</a></li>
-								<li><a href="UmbracoCms.4.8.1.zip" rel="download">4.8.1</a></li>
-								<li><a href="UmbracoCms.4.9.0.zip" rel="download">4.9.0</a></li>
-								<li><a href="UmbracoCms.4.9.1.zip" rel="download">4.9.1</a></li>
+								<li>
+									<a href="UmbracoCms.4.7.2.zip" rel="download">4.7.2</a>
+									<span class="commit">
+										(<a href="https://github.com/umbraco/Umbraco-CMS/releases/tag/4.7.2" rel="external" target="_blank">See release on GitHub</a>)
+									</span>
+								</li>
+								<li>
+									4.8.0 beta

```

Comment:
Personally I won't need them, but I added them since I see people on Twitter and Our using beta and release candidates even for production environments. But hopefully it is only a fraction of Umbraco users that do so.